lib.php

```diff
@@ -221,13 +213,6 @@ function accredible_update_instance($post) {
         }
     }
 
-    // If the group was changed we should save that.
-    if (!$accrediblecertificate->achievementid && $post->groupid) {
-        $groupid = $post->groupid;
-    } else {
-        $groupid = $accrediblecertificate->groupid;
-    }
-
```

Comment:
`achievementid` is related to the legacy features and I have not fully understood how they work so I'd rather keep this logic just in case.

Reply:
I have added this code again.